xml/art_pci-dss.xml

```diff
@@ -0,0 +1,49 @@
+<?xml version="1.0" encoding="utf-8"?>
+<?xml-stylesheet href="urn:x-suse:xslt:profiling:docbook50-profile.xsl"
+ type="text/xml"
+ title="Profiling step"?>
+
+<!DOCTYPE article
+[
+  <!ENTITY % entities SYSTEM "generic-entities.ent">
+  %entities;
+  <!ENTITY productnamex "&sls;">
+  <!ENTITY productnumberx "&product-ga; SP&product-sp;">
+]>
+
+<article xml:id="article-pci-dss" xml:lang="en" version="5.0"
+  xmlns="http://docbook.org/ns/docbook"
+  xmlns:xi="http://www.w3.org/2001/XInclude"
+  xmlns:xlink="http://www.w3.org/1999/xlink">
+ <title>&sle_pcidss_guide;</title>
+ <titleabbrev>&sle_pcidss_guide;</titleabbrev>
+ <info>
+  <productname>&productnamex;</productname>
+  <productnumber>&productnumberx;</productnumber>
+  <date><?dbtimestamp format="B d, Y" ?></date>
+  <dm:docmanager xmlns:dm="urn:x-suse:ns:docmanager">
+   <dm:bugtracker>
+    <dm:url>https://bugzilla.suse.com/enter_bug.cgi</dm:url>
+    <dm:component>Documentation</dm:component>
+    <dm:product>SUSE Linux Enterprise Server 15 SP4</dm:product>
```

Comment:
Isn't this article supposed to be unversioned?

Reply:
That's a good question. Well, should be, right? However, I just copied it from doc-sle. :slightly_smiling_face:

Reply:
This part is needed to fill our bugtracker. So we can leave it as is for the time being.